src/rules/listValidator.test.js

```
@@ -0,0 +1,89 @@
+        min: 1,
+        max: 3,
+        items: {
+          min: 1,
+          max: 5,
+        },
+      },
+    };
+  });
+
+  it("should return an empty array if template.lists is not defined", () => {
+    delete template.lists;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)